### PR TITLE
Tri wizard

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
Moves colours into `colours.less` and makes proper use of brand primary, secondary and tertiary.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
none

#### Screenshots / Screencast
screen shot 2017-02-21 at 11 59 13
screen shot 2017-02-21 at 11 59 24

#### What gif best describes how you feel about this work?
giphy-2

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

|Globals||
|---|
|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-globals)|

|Brand| Version   |
|---|---|
|Airparks| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-airparks)  |
|Alton Towers| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-altontowers)  |
|Chessington| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-chessington)  |
|Default| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-default)  |
|Essential Travel|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-essential)   |
|Globals|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-globals)   |
|Harry Potter|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-harry-potter)   |
|Holidayextras|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-hx)   |
|Legoland|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-legoland)   |
|Thorpe Park| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-thorpepark)  |
|Legoland|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-legoland)   |
|Peppa Pig| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-peppa)  |
|Legoland|[npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-legoland)   |
|Paultons| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-paultons)  |
|Warwick| [npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-warwick)  |

---

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -26,17 +26,17 @@ By adding a +1 you are confirming you have...
...
 |Globals|
-|---|---|
+|---|
 |[![npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-globals.svg)](https://badge.fury.io/js/%40holidayextras%2Fbrand-globals)|
 
 |Brand| Version   |
 |---|---|
 |Airparks| [![npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-airparks.svg)](https://badge.fury.io/js/%40holidayextras%2Fbrand-airparks)  |
 |Alton Towers| [![npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-altontowers.svg)](https://badge.fury.io/js/%40holidayextras%2Fbrand-altontowers)  |
+|BCP| [![npm version](https://badge.fury.io/js/%40holidayextras%2Fbrand-bcp.svg)](https://badge.fury.io/js/%40holidayextras%2Fbrand-bcp)  |
```
Comment (nickine) [inline .github/PULL_REQUEST_TEMPLATE.md]: is an this intentional add?

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/harry-potter/src/less/variables.less [hunk 2]
```diff
@@ -5,20 +5,11 @@
...
-@brand-danger:          @color-red;
-
-
 //== Scaffolding
 //
-@body-bg:               @color-cream-lighter;
-@link-color:            @color-brown-light;
-@text-color-inverse:    @color-cream-lighter;
+@body-bg:               @brand-secondary-lighter;
+@link-color:            @color-blue;
```
Comment (nickine) [inline brands/harry-potter/src/less/variables.less]: blue?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jodiedoubleday) [inline .github/PULL_REQUEST_TEMPLATE.md]: shhhhh

File: brands/harry-potter/src/less/variables.less (hunk 2 above)
Comment (jodiedoubleday) [inline brands/harry-potter/src/less/variables.less]: well... they don't actually have red links - https://www.harrypottertheplay.com/ticket-information/

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (nickine) [inline .github/PULL_REQUEST_TEMPLATE.md]: well well well

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Pull request merged**